README.md

```diff
@@ -1,218 +1,180 @@
-```
-Usage: ankerctl.py [OPTIONS] COMMAND [ARGS]...
-
-Options:
-  -k, --insecure  Disable TLS certificate validation
-  -v, --verbose   Increase verbosity
-  -q, --quiet     Decrease verbosity
-  -h, --help      Show this message and exit.
-
-Commands:
-  config  View and update configuration
-  http    Low-level http api access
-  mqtt    Low-level mqtt api access
-  pppp    Low-level pppp api access
-```
+> **Warning**
+> Docker Installation ONLY works on Linux at this time
 
-Before you can use ankerctl, you need to import the configuration.
+Follow the instructions for a [git install](documentation/install-from-git.md) (recommended), or [docker install](documentation/install-from-docker.md).
 
-Support for logging in with username and password is not yet supported, but an
-`auth_token` can be imported from the saved credentials found in `login.json` in
-Ankermake Slicer. See `ankerctl.py config import -h` for details:
+## Importing configuration
 
-```
-Usage: ankerctl.py config import [OPTIONS] path/to/login.json
+1. Import your AnkerMake account data by opening a terminal window in the folder you placed ankerctl in and running the following command:
 
-  Import printer and account information from login.json
+   ```sh
+   ./ankerctl.py config import [OPTIONS] path/to/login.json 
+   ```
 
-  When run without filename, attempt to auto-detect login.json in default
-  install location
+   When run without filename on Windows and MacOS, the default location of `login.json` will be tried if no filename is specified. Otherwise, you can specify the file path for `login.json`. Example for Linux:
+   ```sh
+   ./ankerctl.py config import ~/.wine/drive_c/users/username/AppData/Local/AnkerMake/AnkerMake_64bit_fp/login.json
+   ```
+   MacOS
+   ```sh
+   ./ankerctl.py config import $HOME/Library/Application\ Support/AnkerMake/AnkerMake_64bit_fp/login.json
+   ```
+   Windows
+   ```sh
+   python3 ankerctl.py config import %APPDATA%\AnkerMake\AnkerMake_64bit_fp\login.json
+   ```
 
-Options:
-  -h, --help  Show this message and exit.
-```
+   Type `ankerctl.py config import -h` for more details on the import options. Support for logging in with username and password is not yet supported. To learn more about the method used to extract the login information and add printers, see the [MQTT Overview](documentation/developer-docs/mqtt-overview.md) and [Example Files](documentation/developer-docs/example-file-usage) documentation.
 
-On Windows and MacOS, the default location of `login.json` will be tried if no
-filename is specified. Otherwise, you can specify the file path for
-`login.json`. Example for linux:
+   The output when successfully importing a config is similar to this:
 
-```sh
-ankerctl.py config import ~/.wine/drive_c/users/username/AppData/Local/AnkerMake/AnkerMake_64bit_fp/login.json
-```
+   ```
```

Comment:
I recommend using ````bash` here for colorful output :)

Reply:
added the color, PM won't say no to pretty.